config/environments/development.rb

```diff
@@ -22,17 +22,9 @@
   # Raise an error on page load if there are pending migrations.
   config.active_record.migration_error = :page_load
 
-  # Debug mode disables concatenation and preprocessing of assets.
-  # This option may cause significant delays in view rendering with a large
-  # number of complex assets.
-  config.assets.debug = true
-
-  # Adds additional error checking when serving assets at runtime.
-  # Checks for improperly declared sprockets dependencies.
-  # Raises helpful error messages.
-  config.assets.raise_runtime_errors = true
-
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # TODO: Move this to config gem / shared_configs
   config.purl_fetcher_url = ''
```

Comment:
Once this gets merged, it allows https://github.com/sul-dlss/discovery-dispatcher/issues/44 to get going. That will allow deployments to set per server